composer.json

```
@@ -7,14 +7,14 @@
         "php": "^7.3 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
         "silverstripe/recipe-cms": "^4",
-        "silverstripe/blog": "3.x-dev",
-        "silverstripe/widgets": "2.x-dev",
-        "silverstripe/content-widget": "2.x-dev",
-        "silverstripe/spamprotection": "3.x-dev",
-        "silverstripe/akismet": "4.x-dev",
-        "silverstripe/comments": "3.x-dev",
-        "silverstripe/comment-notifications": "2.x-dev",
-        "colymba/gridfield-bulk-editing-tools": "^3.0@beta"
+        "silverstripe/blog": "3.9.x-dev",
+        "silverstripe/widgets": "2.2.x-dev",
+        "silverstripe/content-widget": "2.3.x-dev",
+        "silverstripe/spamprotection": "3.2.x-dev",
+        "silverstripe/akismet": "4.2.x-dev",
+        "silverstripe/comments": "3.8.x-dev",
```

```suggestion
        "silverstripe/comments": "3.7.x-dev",
```

We shipped with 3.7.2

---

Good catch, have updated